tools/topology/sof-apl-keyword-detect.m4

```diff
@@ -0,0 +1,102 @@
+#
+# Topology for ApolloLake with direct attach digital microphones array for
+# keyword detection and triggering use case.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
+include(`platform/intel/bxt.m4')
+include(`platform/intel/dmic.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM 0 <-------+- KPBM 0 <-- B0 <-- DMIC6 (DMIC01)
+#               |
+# Keyword <-----+
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     frames, deadline, priority, core)
+
+# Passthrough capture pipeline 1 on PCM 0 using max 2 channels.
+# Schedule 16 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-kfbm-capture.m4,
+	1, 0, 2, s32le,
+	16, 1000, 0, 0)
```

Comment:
320 frames for 20 ms periods.

Reply:
We haven't verified that if 320 frames(20ms period) work, so I prefer to update it later.

Reply:
please make that value configurable without delay so we don't have to comb through the topology later...even if you keep the same value for now the ask is that with a single line change all the topology uses the SAME value. Makes sense?

Reply:
sure, I am waiting for @tlauda's confirmation if we do need 320 frames for the host capture pipeline.

Reply:
@keyonjie We need 320 frames for both kwd pipelines. If by host capture you mean the one with dai, kpb and host components, then yes.